go/vt/vtorc/db/tls.go

```diff
@@ -124,15 +124,15 @@ func SetupMySQLTopologyTLS(uri string) (string, error) {
 }
 
 // Create a TLS configuration from the config supplied CA, Certificate, and Private key.
-// Register the TLS config with the mysql drivers as the "orchestrator" config
+// Register the TLS config with the mysql drivers as the "vtorc" config
 // Modify the supplied URI to call the TLS config
-func SetupMySQLOrchestratorTLS(uri string) (string, error) {
-	if !orchestratorTLSConfigured {
+func SetupMySQLVTOrcTLS(uri string) (string, error) {
+	if !vtorcTLSConfigured {
```

Comment:
I am observing all the flags and function name are VTOrc then why is vtorcTLSConfigured not following this convention?